src/hotspot/share/gc/shenandoah/shenandoahGeneration.cpp

```diff
@@ -209,9 +215,10 @@ void ShenandoahGeneration::merge_write_table() {
 }
 
 void ShenandoahGeneration::prepare_gc() {
-  // Reset mark bitmap for this generation (typically young)
-  reset_mark_bitmap();
```

Comment:
Is `reset_mark_bitmap()` dead code and therefore removable now? Or, can all other usages also be removed in similar manner?

Reply:
No, it's not dead. It's used by the full GC still.